### ISSUE TITLE
Missing package

### ISSUE THREAD

**Issue by xiu-ccc** [o2as/ur-o2as#6]
Excuse me, I am trying to run `roslaunch o2as_examples example_tf_moveit.launch` and found that there is no such launch file under  o2as_examples package.Can you provide this file？

**Comment by felixvd** [comment]
Looks like it's called `example_tf.launch` now and the documentation was outdated. Can you confirm?

**Comment by xiu-ccc** [comment]
I am trying to run `example_tf.launch`  ,but the following error appeared：
```
[ INFO] [1625293950.274391086, 10.706000000]: Ready to take commands for planning group a_bot.
terminate called after throwing an instance of 'tf2::LookupException'
  what():  "set2_bin1" passed to lookupTransform argument target_frame does not exist. 
[ExampleLoopMoveit-1] process has died [pid 19303, exit code -6, cmd /root/catkin_ws/devel/lib/o2as_examples/example_tf __name:=ExampleLoopMoveit __log:=/root/.ros/log/0b508eb2-dafa-11eb-ac42-f40669541898/ExampleLoopMoveit-1.log].
log file: /root/.ros/log/0b508eb2-dafa-11eb-ac42-f40669541898/ExampleLoopMoveit-1*.log
all processes on machine have died, roslaunch will exit
shutting down processing monitor...
... shutting down processing monitor complete
done
```

**Comment by felixvd** [comment]
True. Those are frames from the kitting scene, so you might have to switch to that scene in `o2as_base_scene.urdf.xacro`. But this example might just be outdated.

I don't have the cycles to fix this myself, but I can merge a PR if you submit a fix.

**Comment by xiu-ccc** [comment]
Excuse me, I want to reproduce the task of kitting. I am also doing some attempts, but I don't have a real machine, I only implement it in a virtual environment. 
I found that you seem to use two different capture methods. One uses the `omron_cad_matching` function, but I don’t have a license. Does this package need to encrypt the USB before it can continue to run?
 At the same time, I also tried the method of `grabability_estimation_server`. I can run normally and show that it starts normally, but there is no response in the virtual environment. I would like to ask you for advice, thank you.

**Comment by felixvd** [comment]
Yes, the vision library was proprietary and is not part of this repository.

I have not used that function and visualization in years, so I cannot be of much help, but if you are publishing visualization markers you should confirm that your Rviz display is active and subscribed to the correct topic. You will find more information about visualization in ROS with those keywords.
